src/test/java/net/minecraftforge/debug/EnchantmentLevelEventTest.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.debug;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.living.LivingEnchantmentLevelEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid="enchantmentleveleventtest",version="1.0")
+public class EnchantmentLevelEventTest
+{
+
+    @Mod.EventHandler
+    public void preinit(FMLPreInitializationEvent evt)
+    {
+        MinecraftForge.EVENT_BUS.register(this);
+    }
+
+    @SubscribeEvent
+    public void onInteract(LivingEnchantmentLevelEvent event)
+    {
+        if (event.getEntityLiving().isSneaking())
+        {
+            int oldLevel = event.getLevel();
+            int newLevel = (oldLevel + 1) * 10;
+            event.setLevel(newLevel);
+            System.out.println("Set fortune level from " + oldLevel + " to " + newLevel);
```

Comment:
The print mentions fortune, but this would still be called for all enchantments.

Reply:
This should use a logger like the other test mods, not `System.out`